orderer/consensus/smartbft/verifier.go

```diff
@@ -190,11 +190,13 @@ func (v *Verifier) verifyRequest(rawRequest []byte, noConfigAllowed bool) (types
 	}
 
 	if req.chHdr.Type == int32(cb.HeaderType_CONFIG) {
-		err := v.ConfigValidator.ValidateConfig(req.envelope)
+		err = v.ConfigValidator.ValidateConfig(req.envelope)
 		if err != nil {
 			v.Logger.Errorf("Error verifying config update: %v", err)
 			return types.RequestInfo{}, err
 		}
+
+		return v.ReqInspector.RequestID(rawRequest), nil
```

Comment:
maybe check if it returns the zero value (empty request ID) then return error?

Reply:
fix